app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss

```diff
@@ -0,0 +1,88 @@
+.related-articles{
+	background-color: #f8f8f8;
+	margin-top: 53px;
+	padding-top: 33px;
+	padding-bottom: 18px;
+
+	&__item-wrapper{
+		@media only screen and (max-width: 660px){
+			display: flex;
+			flex-direction: column;
+		}
+
+		@media only screen and (max-width: 560px){
+			align-items: center;
+		}
+	}
+
+	&__item{
+		width: 24%;
+		display: inline-block;
+		margin-right: 46px;
+
+		&:last-child{
+			margin-right: 0;
+		}
+
+		&:hover .related-articles__banner-img{
+			opacity: .8;
+		}
+
+		@media only screen and (max-width: 920px){
+			width: 30%;
+			margin-right: 4%;
+		}
+
+		@media only screen and (max-width: 660px){
+			width: 50%;
+			margin-right: 0;
+			margin-bottom: 20px;
+		}
+
+		@media only screen and (max-width: 560px){
+			width: 65%;
+		}
+
+		@media only screen and (max-width: 480px){
+			width: 75%;
+		}
+
+		@media only screen and (max-width: 375px){
+			width: 85%;
+		}
+	}
+
+	&__banner-img{
+		background-image: image-url("bg.jpg");
+		background-size: cover;
+		background-position: center;
+		height: 179px;
+		border-radius: 15px;
+		transition: all .3s ease-in-out;
+	}
+
+	&__title{
+		margin: 0;
```

Comment:
i think you need more gap here, check the mockup

Reply:
sorry im not focused